pkgs/test_core/lib/src/direct_run.dart

```diff
@@ -80,7 +83,7 @@ Future<bool> _directRunTests(FutureOr<void> Function() testMain,
       throw MissingTestException(fullTestName);
     }
   }
-  return success!;
+  return success;
```

Comment:
Is it worth keeping a `/*!*/` comment here?

Reply:
I don't think so, they should just get inserted by the migration tool. I didn't actually realize I could... but going back and fixing them would be pretty cumbersome at this point lol.

Reply:
SGTM. We have the git history if we need it.